Gemfile

```diff
@@ -48,12 +50,5 @@ group :test do
   gem 'coveralls', require: false
 end
 
-gem 'capybara', group: [:development, :test]
-gem 'launchy', group: [:development, :test]
-gem 'factory_girl', group: [:development, :test]
-gem 'valid_attribute', group: [:development, :test]
-gem 'shoulda-matchers', group: [:development, :test], require: false
 gem 'devise'
 gem 'foundation-rails'
-
-ruby "2.0.0"
```

Comment:
I would specify the version of Ruby you want in your Gemfile. Heroku uses this when you deploy.